### ISSUE TITLE
Latest Vulkan-Headers breaks VL build

### ISSUE THREAD

**Issue by nsubtil** [KhronosGroup/Vulkan-ValidationLayers#681]
Commit KhronosGroup/Vulkan-Headers@7f02d9bb810f371de0fe833c80004c34f7ff8c57 breaks the VL build in a non-obvious way:

```
[build] Vulkan-ValidationLayers\layers\parameter_validation.cpp(131): error C2440: 'initializing': cannot convert from 'initializer list' to 'std::vector<VkFilter,std::allocator<_Ty>>'
[build]         with
[build]         [
[build]             _Ty=VkFilter
[build]         ]
[build] Vulkan-ValidationLayers\layers\parameter_validation.cpp(131): note: No constructor could take the source type, or constructor overload resolution was ambiguous
```

This happens because the IMG_filter_cubic extension introduces a `#define VK_FILTER_CUBIC_EXT 1000015000` statement, which causes the AllVkFilters vector declaration to resolve to:

```const std::vector<VkFilter> AllVkFilterEnums = {VK_FILTER_NEAREST, VK_FILTER_LINEAR, 1000015000, };```

I'm not sure if this is a bug in the headers, spec or VL itself, but since it breaks VL I'm filing a bug here to start with. From an initial check, I'm surprised that we're using #define on a token that was previously defined as an enum, but I think this would still generate valid code in most contexts --- as far as I can tell, this breaks VL because the number of arguments happens to make ctor resolution ambiguous.

**Comment by karl-lunarg** [comment]
It seems to be an error in the registry.

https://github.com/KhronosGroup/Vulkan-Docs/pull/912
https://github.com/KhronosGroup/Vulkan-Headers/issues/40

**Comment by jzulauf-lunarg** [comment]
@shannon-lunarg please assess and If there is nothing to fix in VVL, close.

**Comment by oddhack** [comment]
Should be fixed in the 1.1.100 Vulkan-Docs spec update.
